CoreScriptsRoot/Modules/Settings/Pages/ResetCharacter.lua

```diff
@@ -7,23 +7,36 @@
 
 -------------- CONSTANTS -------------
 local RESET_CHARACTER_GAME_ACTION = "ResetCharacterAction"
+local RESET_ENABLED_TEXT = "Are you sure you want to reset your character?"
+local RESET_DISABLED_TEXT = "The game doesn't allow you to reset!"
```

Comment:
Yeah, something like "You can currently not reset!" is maybe better.
That's something that can always be changed when needed.